### PR TITLE
PIL-113 Add option to delete profile picture

### PR DESCRIPTION
**Pull request by graszka22**

### CODE REVIEW

File: src/services/api.js [hunk 1]
```diff
@@ -379,6 +379,16 @@ class SDKWrapper {
...
   }
 
+  deleteUserAvatar(walletId: string): Promise<boolean> {
+    return Promise.resolve()
+      .then(() => this.pillarWalletSdk.user.deleteProfileImage({ walletId }))
+      .then(response => response.status === 204)
+      .catch((error) => {
+        reportLog('Failed to delete user avatar', { error }, Sentry.Severity.Error);
+        return false;
+      });
```
Comment (IAmKio) [inline src/services/api.js:389]: Just a comment, if the return type of this function is already a Promise, do we need return Promise.resolve()? For example, could the function be written like this:

```suggestion
  deleteUserAvatar(walletId: string): Promise<boolean> {
    await this.pillarWalletSdk.user.deleteProfileImage({ walletId }))
      .then(response => response.status === 204)
      .catch((error) => {
        reportLog('Failed to delete user avatar', { error }, Sentry.Severity.Error);
        return false;
      };
```

**Review by IAmKio — COMMENTED** [review]
Just a small comment regarding a Promise'd function, other than that - looks good.

File: src/services/api.js (hunk 1 above)
Comment (graszka22) [reply src/services/api.js:389]: It doesn't really matter but it's a convention in this file. Each api function starts with Promise.resolve(). Maybe code aesthetic?

Comment (IAmKio) [reply src/services/api.js:389]: No problem - not going to cry about this one 😂

**Review by IAmKio — APPROVED** [review]

**Pull request merged**